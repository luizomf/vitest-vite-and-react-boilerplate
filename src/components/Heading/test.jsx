import { render, screen } from '@testing-library/react';
import { expect, it } from 'vitest';
import { Heading } from '.';

it('should render Heading with text Hello YouTube', () => {
  render(<Heading>Hello YouTube!</Heading>);
  const heading = screen.getByRole('heading', { name: /Hello YouTube/ });
  expect(heading).toBeInTheDocument();
  expect(heading).toMatchSnapshot();
});
