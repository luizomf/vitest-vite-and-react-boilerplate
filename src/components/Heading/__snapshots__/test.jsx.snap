// Vitest Snapshot v1

exports[`should render Heading with text Hello YouTube 1`] = `
<h1>
  Hello YouTube!
</h1>
`;
